tests/templates/test_subroutines/test_qdrift.py

```diff
@@ -0,0 +1,239 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for the QDrift template.
+"""
+import copy
+import pytest
+from functools import reduce
+
+import pennylane as qml
+from pennylane import numpy as qnp
+from pennylane.math import allclose, get_interface
+from pennylane.templates.subroutines.qdrift import _sample_decomposition
+
+
+test_hamiltonians = (
+    (
+        [1, 1, 1],
+        [qml.PauliX(0), qml.PauliY(0), qml.PauliZ(1)],
+    ),
+    (
+        [1.23, -0.45],
+        [qml.s_prod(0.1, qml.PauliX(0)), qml.prod(qml.PauliX(0), qml.PauliZ(1))],
+    ),  # op arith
+    (
+        [1, -0.5, 0.5],
+        [qml.Identity(wires=[0, 1]), qml.PauliZ(0), qml.PauliZ(1)],
+    ),
+)
+
+
+def mocked_choice(*args, **kwargs):
+    """Fix randomness"""
```

Comment:
Is it a doc for this function or a note for the line after?

Reply:
Removed